src/Illuminate/Support/Str.php

```diff
@@ -87,7 +87,7 @@ public static function contains($haystack, $needles)
     public static function endsWith($haystack, $needles)
     {
         foreach ((array) $needles as $needle) {
-            if ((string) $needle === substr($haystack, -strlen($needle))) {
+            if ((string) $needle === mb_substr($haystack, -mb_strlen($needle))) {
```

Comment:
You can use static method
mb_substr --> static::substr
mb_strlen --> static::length